foundations/html_css/the-box-model/the-box-model.md

```diff
@@ -46,7 +46,7 @@ Be sure to study the diagrams carefully.
 <div class="lesson-content__panel" markdown="1">
 
 1. [This video](https://www.youtube.com/watch?v=rIO5326FgPE) is a straightforward overview of the box model, padding and margin. Go ahead and watch this now; it informs everything else.
-2. Because the box model concept is so incredibly fundamental, check out [this lesson from MDN](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model). It covers the same material as the video above, but it goes a little further in-depth. Pay close attention to the examples and take the time to experiment with their in-browser editor!
+2. Because the box model concept is so incredibly fundamental, check out [these sections from MDN box model lesson](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model#what_is_the_css_box_model). Read up to [Margins, padding, and borders](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model#margins_padding_and_borders). It covers the same material as the video above, but it goes a little further in-depth. Pay close attention to the examples and take the time to experiment with their in-browser editor!
```

Comment:
links doesn't work on mobile browsers

The upcoming lesson talks about these concepts from the very beginning with no confusion

 I made another commit #25476 referencing the same article

Reply:
The links are not working for me on a computer. Well, to be clear the link works, but it does not take me to the heading that the link indicates that it should. Every link takes me to the same place.

Reply:
Hey @rlmoser99, Thanks for your patience with me. I'm sorry for the confusing here. I tried these links on both FF and chromium. They work as expected. These links are clearly valid links from MDN with heading ID.

Reply:
Heading IDs for MDN do not seem to work in the following browser: Chrome, Edge, Opera. They do work in Firefox however.

Reply:
@i4pg Because the inconsistency of the links to this awesome resource, I do not think we should split up the reading between two lessons. Instead, I suggest that he just provide a "warning" that they will be introduced to new material (previously it somewhat implies that all of the material was previously covered). So something like this:

```suggestion
2. Because the box model concept is so incredibly fundamental, let's dig a bit deeper with [this lesson from MDN](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model). It covers the same material as the video above and will introduce you to inline boxes that we will explore in the next lesson. Pay close attention to the examples and take the time to experiment with their in-browser editor!
```

Reply:
That seems fine. It will help students prepare for what's coming in this assignment. Thanks.